src/joint.dia.cell.js

```diff
@@ -980,6 +980,7 @@ joint.dia.CellView = Backbone.View.extend({
     pointermove: function(evt, x, y) {
 
         this.notify('cell:pointermove', evt, x, y);
+
```

Comment:
If `joint.dia.cell.js` doesn't have to be changed after all. Can you please remove this extra lines?
And once everything else is finished and ready for merging to the upstream can you merge all your commits into a one so we keep the history clearer and better to read?

    git rebase -i HEAD~[number-of-commits]